clippy_utils/src/lib.rs

```diff
@@ -1253,22 +1255,33 @@ pub fn get_item_name(cx: &LateContext<'_>, expr: &Expr<'_>) -> Option<Symbol> {
     }
 }
 
-pub struct ContainsName {
+pub struct ContainsName<'a, 'tcx> {
+    pub cx: &'a LateContext<'tcx>,
     pub name: Symbol,
     pub result: bool,
 }
 
-impl<'tcx> Visitor<'tcx> for ContainsName {
+impl<'a, 'tcx> Visitor<'tcx> for ContainsName<'a, 'tcx> {
+    type NestedFilter = nested_filter::All;
```

Comment:
yeah i think it should be OnlyBodies here, All includes things like impls, etc, whereas OnlyBodies just covers closures and consts.

We actually don't want to include consts but that's niche enough and it doesn't hurt to check extra

Reply:
👍 Thanks. I made a change to use `nested_filter::OnlyBodies` instead.